.changeset/plenty-laws-attend/changes.md

```diff
@@ -0,0 +1 @@
+Makes the password auth strategy its own package
```

Comment:
Could you add a short explanation/code example of how consumers should migrate?

Reply:
Sure, done :)